Analyzer/interface/CommonFunction.h

```diff
@@ -1174,7 +1174,7 @@ bool clusterCleaning(std::vector<int> ampls, int crosstalkInv = 0, uint8_t* exit
       MaxOnEnd = true;
     }
   }
-  // Si une seule strip touch�e
+  // Si une seule strip touchée
```

Comment:
Can you actually translate that to english please? :)

Reply:
sure!

Reply:
I'll translate all sentences to english